### DISCUSSION TITLE
OQS Openssl 3

### DISCUSSION THREAD

**Discussion by randheerar** [open-quantum-safe/openssl#436] [General]
Thanks for awesome work being done here. Wondering, is there a stable release planned for OQS Openssl v3?

**Comment by baentsch** [comment]
May I ask why you'd like to see this? IMO there is no need for such OQS-specific fork of OpenSSLv3: Plain, vanilla OpenSSLv3(.2) can perform PQ crypto when loading our [oqs-provider](https://github.com/open-quantum-safe/oqs-provider) plugin. [Sample integrations of oqs-provider+OpenSSLv3](https://github.com/open-quantum-safe/oqs-demos/issues/182) already show this works nicely in downstream software.
